tests/XrdPfcTests/XrdPfcTests.cc

```diff
@@ -0,0 +1,19 @@
+
+#include "XrdPfc/XrdPfcPathParseTools.hh"
+
+#include <gtest/gtest.h>
+
+using namespace XrdPfc;
+
+TEST(XrdPfcTests, PathTokenizer)
+{
+    PathTokenizer("/pelican/hello_world.txt", 0, true);
```

Comment:
I think this would be a lot simpler if you used loops (see XrdClURL test).

Reply:
Yes - I assume that Alja and Matevz will expand it into a larger test by pushing to the branch and not using as-is.  Just wanted to help show the crash.